lib/inline_svg/io_resource.rb

```diff
@@ -3,14 +3,7 @@
 module InlineSvg
   module IOResource
     def self.===(object)
-      object.is_a?(IO) || object.is_a?(StringIO)
-    end
-
-    def self.default_for(object)
-      case object
-      when StringIO then ''
-      when IO then 1
-      end
+      object.is_a?(IO) || object.is_a?(StringIO) || object.is_a?(Tempfile)
     end
```

Comment:
hello, I understand that `default_for` appears not to be used by this library and it is not even tested, but is there a reason to remove it entirely?

Reply:
Great, I think it's a great option. As soon as the other PR is up, I'll update this one!